Lib/idlelib/configdialog.py

```diff
@@ -88,6 +88,7 @@ def __init__(self, parent, title='', _htest=False, _utest=False):
         self.attach_var_callbacks()  # Avoid callbacks during load_configs.
 
         if not _utest:
+            self.grab_set()
```

Comment:
I only saw it at `textview` that control with `model` argument. `grab_set` somehow will affect `event_generate`, too.